cpplint_clitest.py

```diff
@@ -41,48 +41,49 @@
 
 from pytest import mark
 from testfixtures import compare
+from parameterized import parameterized
```

Comment:
Move this line just above pytest

Reply:
Yeah. I'm also working on a rebase so that this PR can be merged by rebase rn.